### ISSUE TITLE
Write Unittests for `src/screens/OrganizationPeople/AddMember.tsx`

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#2375]
We will need unittests done for all methods, classes and functions found in this file.

### PR Acceptance Criteria

1. When complete this file must show **100%** coverage when merged into the code base.
1. [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin/tree/develop/?displayType=list)
1. CodeRabbit must approve all your changes to the test file(s) and the file(s) under test
1. The PR will show a report for the code coverage for the file you have added. You can use that as a guide
1. No functionality must be removed to achieve this goal
1. All tests must be valid and truly test the code base

### Other Criteria

1. If this file isn't referenced in the code base then remove it. We don't want tests run on unused code

### Potential internship candidates

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by sethdivyansh** [comment]
please assign me.

**Comment by ARYANSHAH1567** [comment]
Can you please assign it to me?

**Comment by Kunalpal216** [comment]
If there are no updates by @ARYANSHAH1567 , Can this issue be assigned to me ?

**Comment by ARYANSHAH1567** [comment]
I am working on it

**Comment by palisadoes** [comment]
During the week of November 11, 2024 we will start a code freeze on the `develop` branches in Talawa, Talawa Admin and Talawa-API. 

We have completed a project to convert the Talawa-API backend to use PostgreSQL. Work will then begin with us merging code in the `develop` branches to a new `develop-postrgres` branch in each repository.

Planning activities for this will be managed in our #talawa-projects slack channel. A GitHub project will be created to track specially labeled issues. We completed a similar exercise last year using a similar methodology. 

Starting November 12, California time no new PRs will be accepted against the `develop` branch. They must be applied to the `develop-postrgres` branch. 

There are some GSoC project features that will need to be merged into develop. These will be the only exceptions.

This activity and the post GSoC 2024 start date was announced in our #general Slack channel last month as a pinned post.

**Comment by Cioppolo14** [comment]
@ARYANSHAH1567 Are you working on this?

**Comment by ARYANSHAH1567** [comment]
> @ARYANSHAH1567 Are you working on this?

No, Actually i am a little busy and i think it would take quite a bit time because there are some tests that are always true and not checking anything. So for now i would like to unassign this

**Comment by palisadoes** [comment]
@PAVAN507-STAR 

Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by Krishnaidnani** [comment]
can you please assign it to me?

**Comment by abbi4code** [comment]
@palisadoes, this PR #3133 includes tests I have written for both organizationpeople.tsx and addmember.tsx, with 100% test coverage, as they are in the same file. Since I am not aware of this existing issue,

**Comment by palisadoes** [comment]
Thanks. Closing this issue
